plugins/extractors/bigquery/bigquery.go

```diff
@@ -284,7 +311,8 @@ func (e *Extractor) buildAsset(ctx context.Context, t *bigquery.Table, md *bigqu
 			"dataset":             t.DatasetID,
 			"project":             t.ProjectID,
 			"type":                string(md.Type),
-			"partition_field":     partitionField,
+			"partition_data":      partitionData,
```

Comment:
Can we please add documentation for fields that are being extracted into attributes?

Reply:
I will add.

Reply:
Done